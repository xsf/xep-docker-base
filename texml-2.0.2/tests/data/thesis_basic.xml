<TeXML>
 <cmd name="documentclass">
  <opt>a4paper</opt>
  <parm>minimal</parm>
 </cmd>
 <env name="document">
  Eto&#xa0;&#x2014; primer.
 </env>
</TeXML>
