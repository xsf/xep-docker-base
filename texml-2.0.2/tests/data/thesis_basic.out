\documentclass[a4paper]{minimal}
\begin{document}
Eto~\textemdash{} primer.
\end{document}
